### PR TITLE
cleanup from clang warnings

### PR DESCRIPTION
**Pull request by adizaimi**

### CODE REVIEW

**Comment by akshatsikarwar** [comment]
Please do not merge this.

File: db/sqlinterfaces.c [hunk 1]
```diff
@@ -950,7 +950,8 @@ int get_high_availability(struct sqlclntstate *clnt)
 int request_durable_lsn_from_master(bdb_state_type *bdb_state, uint32_t *file,
                                     uint32_t *offset, uint32_t *durable_gen);
 
-static int fill_snapinfo(struct sqlclntstate *clnt, int *file, int *offset)
+static int fill_snapinfo(struct sqlclntstate *clnt, unsigned int *file,
+                         int *offset)
```
Comment (ldennis) [inline db/sqlinterfaces.c]: double check what type is offset?

File: berkdb/txn/txn.c [hunk 2]
```diff
@@ -580,7 +580,7 @@ __txn_begin_int_int(txn, internal, retries, we_start_at_this_lsn)
 	txn->id = __txn_id;
 	txn->prepare = __txn_prepare;
 	txn->set_timeout = __txn_set_timeout;
-	txn->snapshot = __txn_snapshot;
+	/* txn->snapshot = __txn_snapshot; */
```
Comment (ldennis) [inline berkdb/txn/txn.c]: also remove __txn_snapshot?

**Comment by ldennis** [comment]
looks good

**Comment by adizaimi** [comment]
thank you.

**Pull request merged**